### PR TITLE
Hide Remove filtered reports button

### PR DESCRIPTION
**Pull request by csordasmarton**
* Show `Remove filtered reports` button at the bottom of the filter bar.
* Hide `Remove filtered reports` button from the filter bar if the user doesn't have *storage* permission for the current product.
* Show the *number of results* which will be deleted on the confirmation dialog.

### CODE REVIEW

**Comment by bruntib** [comment]
This report removal works for me. I have just one note: when the reports are removed, then the filter options remain the same. This results that I get an empty page, since the filter options refer to the deleted reports. Shouldn't the filter options be reset to the default options except for the run name?

**Comment by whisperity** [comment]
I've been out of the loop regarding CodeChecker &mdash; is this a webapp only feature?

File: www/style/codecheckerviewer.css [hunk 1]
```diff
@@ -14,6 +14,10 @@ html, body {
   display: none !important;
 }
 
+.error {
```
Comment (whisperity) [inline www/style/codecheckerviewer.css]: An error colour has already been introduced once with the product management interface - perhaps it should be revised or reused?

**Comment by csordasmarton** [comment]
@whisperity Yes, this is only a web app feature. We discussed it with @gyorb that the user must see what what kinds of reports will be removed.

File: www/style/codecheckerviewer.css (hunk 1 above)
Comment (csordasmarton) [reply www/style/codecheckerviewer.css]: I didn't find any definition of this class in the css files. I only found these:
https://github.com/Ericsson/codechecker/blob/d94f08a9fe78e7618b78c818e33000a610336a63/www/style/codecheckerviewer.css#L208
https://github.com/Ericsson/codechecker/blob/d94f08a9fe78e7618b78c818e33000a610336a63/www/style/codecheckerviewer.css#L237

Comment (whisperity) [reply www/style/codecheckerviewer.css]: Later down in the file there is `mbox mbox-error`. Those are used in the product manager interface for reporting back if e.g. the server rejects your database settings.

**Pull request merged**